vector/src/main/java/im/vector/app/features/rageshake/BugReportActivity.kt

```diff
@@ -60,30 +63,48 @@ class BugReportActivity : VectorBaseActivity<ActivityBugReportBinding>() {
             views.bugReportButtonIncludeScreenshot.isEnabled = false
         }
 
-        forSuggestion = intent.getBooleanExtra("FOR_SUGGESTION", false)
+        reportType = intent.getStringExtra(REPORT_TYPE_EXTRA)?.let {
+            tryOrNull { ReportType.valueOf(it) }
+        } ?: ReportType.BUG_REPORT
 
         // Default screen is for bug report, so modify it for suggestion
-        if (forSuggestion) {
-            supportActionBar?.setTitle(R.string.send_suggestion)
+        when (reportType) {
+            ReportType.BUG_REPORT -> {
+                supportActionBar?.setTitle(R.string.title_activity_bug_report)
+                views.bugReportButtonContactMe.isVisible = false
```

Comment:
Why not keeping this new option also for bug report?

Reply:
ok